content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -725,28 +725,30 @@ stages:
       feature: [feature1, feature2, feature3]
     cmd: ./train.py --feature ${item.feature} ${item.model}
     outs:
-      - ${item.model}.pkl
+      - ${key}.pkl # equivalent to: ${item.model}-${item.feature}.pkl
 ```
 
 You can reference each variable in your stage definition using the `item`
 dictionary key. In the above example, you can access `item.model` and
-`item.feature`.
+`item.feature`. Moreover, `matrix` exposes a `key` value, which combines the
+current `item` values into one expression (equal to the current stage name after
+the `@` – see example below).
```

Comment:
```suggestion
current `item` values into one expression (see example below).
```

You don't have to accept this suggestion, but I think this part is more confusing than helpful -- seeing the example is more clear than explaining it.